PixiEditor/ViewModels/SubViewModels/Main/DebugViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Diagnostics;
+using PixiEditor.Helpers;
+
+namespace PixiEditor.ViewModels.SubViewModels.Main
+{
+    public class DebugViewModel : SubViewModel<ViewModelMain>
+    {
+        public RelayCommand BreakCommand { get; set; }
+
+        public DebugViewModel(ViewModelMain owner)
+            : base(owner)
+        {
+            BreakCommand = new RelayCommand(Break, CanBreak);
+        }
+
+        public static void Break(object parameter)
```

Comment:
I don't see the point of this menu option, while we can keep this VM, I would remove this Command